components/application-operator/charts/application/templates/deployment.yaml

```diff
@@ -97,8 +97,6 @@ spec:
         - "/eventservice"
         - "--externalAPIPort={{ .Values.eventService.deployment.args.externalAPIPort }}"
         - "--eventMeshURL=http://{{ .Release.Name }}.{{ .Release.Namespace }}"
-        - "--eventsTargetURLV1={{ .Values.eventService.deployment.args.eventsTargetURLV1 }}"
```

Comment:
as far as I can see here https://github.com/kyma-project/kyma/blob/master/components/event-service/cmd/eventservice/options.go#L21 this cmd arg is still valid and used, and if it's not sent, the default is the event mesh new endpoint . am I missing something?

Reply:
yeah, during refactoring eventmesh we might have forgotten this one
i will remove it